### ISSUE TITLE
Configs changes not applying

### ISSUE THREAD

**Issue by tsengunr** [HakonHarnes/img-clip.nvim#87]
Its my config settings down here, but its not even effecting to the plugin. for example file extension type, saving directory name, template file name etc, or am i doing it wrong? please correct me if i am. thanks
```
require('img-clip').setup({
  default = {
    dir_path = "imagesss", ---@type string | fun(): string
    use_absolute_path = false, -- Avoid absolute paths for portability across systems
    relative_to_current_file = true, -- Save images relative to the current file instead of the working directory
    -- dir_path = function() return vim.fn.expand("%:t:r") .. "-img" end, -- Create a directory based on the filename
    prompt_for_file_name = false, -- Automatically name images using a timestamp
    file_name = "%Y-%m-%d-at-%H-%M-%S", -- Format for autogenerated filenames
    extension = "avif", -- Save images as AVIF for better compression
    process_cmd = "convert - -quality 75 avif:-", -- Compress images to 75% quality in AVIF format
  },
  filetypes = {
    markdown = {
      url_encode_path = true, -- Encode special characters in Markdown image paths
      template = "![$FILE_NAME]($FILE_PATH)" -- Markdown template for inserting images
    },
  },
})

```

**Comment by HakonHarnes** [comment]
Your configuration looks fine. Which package manager are you using?

**Comment by HakonHarnes** [comment]
Closing due to inactivity. Let me know if you still need help.

**Comment by tsengunr** [comment]
Hi, thanks for reaching out. Just today i tried changing my nvim config and replaced my package manager with Lazy instead of Packer. But then my long time inactive img-clip config stated working somehow:). i dont know if it was only caused by my Package manager or code etc no idea but in the end im happy now. thank you so much:)

**Comment by HakonHarnes** [comment]
Happy to hear that!

**Comment by proofer** [comment]
I'm having this issue but with lazy.nvim. I'm using default values except I changed `prompt_for_file_name` to be `false`, yet it still prompts for a file name; and I have `relative_to_current_file = true` yet it saves the file in `./accept/` though the current file is several directory levels below the current directory. And while the file it saves does have a `.png` extension ,the file is empty. It does put an image link into the current file, which is a markdown filetype. I checked to make sure the clipboard contains an image.
